worldedit-core/src/main/java/com/sk89q/worldedit/extension/platform/AbstractPlayerActor.java

```diff
@@ -157,6 +151,44 @@ public boolean isHoldingPickAxe() {
                 || item == ItemTypes.NETHERITE_PICKAXE;
     }
 
+    @Override
+    public Direction getCardinalDirection(int yawOffset) {
+        final Location location = getLocation();
+        if (location.getPitch() > 67.5) {
+            return Direction.DOWN;
+        }
+        if (location.getPitch() < -67.5) {
+            return Direction.UP;
+        }
+
+        // From hey0's code
```

Comment:
Who's that? Are we referring something here? Better use `@author` then.

Reply:
Irrelevant as this is from upstream.